<ui version="4.0" >
 <class>yesNoDialog</class>
 <widget class="QWidget" name="yesNoDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Are you sure you want to delete the selected items?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushYes" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushNo" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>70</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
